src/network_models/power_flow_evaluation.jl

```diff
@@ -385,6 +385,8 @@ _get_pf_result(::Type{PowerFlowLineActivePowerFromTo}, pf_data::PFS.PowerFlowDat
     PFS.get_branch_activepower_flow_from_to(pf_data)
 _get_pf_result(::Type{PowerFlowLineActivePowerToFrom}, pf_data::PFS.PowerFlowData) =
     PFS.get_branch_activepower_flow_to_from(pf_data)
+_get_pf_result(::Type{PowerFlowLossFactors}, pf_data::PFS.PowerFlowData) =
+    [PFS.penalty_factors(aux_vars) for aux_vars in pf_data.aux_variables]
```

Comment:
First, a bit of syntactic sugar:
```suggestion
    PFS.penalty_factors.(pf_data.aux_variables)
```

Reply:
Second, maybe the issue you're having here is that this is a vector of vectors rather than a matrix? Might need to do some reshaping (see `hcat`, `vcat`, etc.).

Reply:
Third, it looks like we now have two different types of things called aux variables. That's going to get confusing.